daffodil-core/src/main/scala/org/apache/daffodil/dsom/RuntimePropertyMixins.scala

```diff
@@ -46,6 +46,8 @@ import org.apache.daffodil.schema.annotation.props.gen.DFDLBaseTypeMixin
 import org.apache.daffodil.schema.annotation.props.gen.DFDLSimpleTypeMixin
 import org.apache.daffodil.schema.annotation.props.gen.LengthAGMixin
 import org.apache.daffodil.schema.annotation.props.gen.LengthKind
+import org.apache.daffodil.schema.annotation.props.gen.ChoiceLengthKind
```

Comment:
ChoiceLenghtKind no longer needs to be imported. There are also some other files that the only changes are now imports that can be backed out.

Reply:
Will do.  I feel like I remember getting errors before about unused imports.  Did we change something?

Reply:
Now that you mention, I think you're right. I think maybe when we added support for scala 2.12 we removed some build options that were scala 2.11 only. It looks like in 2.11 the build flag is ``-Ywarn-unused-import`` and scala 2.12 is ``-Ywarn-unused:imports`` (note the colon and the plural). So if we want this back (seems reasonable), we'd need to conditionally add the right option to scalacOptions based on scalaVersion.